ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -62,17 +64,20 @@ def pose_cb(self, msg):
 
     def waypoints_cb(self, msg):
         self.wps = msg.waypoints
+        self.wps_base = copy.deepcopy(self.wps)
         for w in self.wps: w.twist.twist.linear.x = 0
 
     def cruise_trajectory(self, next_wp):
         curr_vel = float(self.wps[next_wp-1].twist.twist.linear.x)
         last_wp = next_wp + LOOKAHEAD_WPS
         wps = self.wps[next_wp : last_wp]
+        wps_base = self.wps_base[next_wp : last_wp]
         # inc_vel = (MAX_VEL - curr_vel)/float(last_wp - next_wp)
         inc_vel = 1.0
```

Comment:
Not sure if a constant acceleration will work always. Should we take the last waypoint's velocity as the max velocity and try to use it to arrive at `inc_vel`? I am not sure what a good solution is ..